package.json

```diff
@@ -29,7 +29,7 @@
     "co-fs": "^1.2.0",
     "coinfalcon": "^1.0.3",
     "commander": "^2.9.0",
-    "gdax": "^0.4.2",
+    "gdax": "github:coinbase/gdax-node",
```

Comment:
I think this requires git to be installed, Gekko's installation (using npm) should also work without Git. I'll check this asap.

Reply:
You might be right, I installed it this way based on information in their official repository.  The command was npm install coinbase/gdax-node. 

I am not sure why they haven't updated the other npm package.